lib/followers/isUserFollowingMultiple.ts

```diff
@@ -0,0 +1,15 @@
+import getSpotifyData from "@/lib/getSpotifyData";
+
+const isUserFollowingMultiple = async (usernames: string) => {
+  const response =
+    await getSpotifyData<SpotifyApi.UserFollowsUsersOrArtistsResponse>(
+      `https://api.spotify.com/v1/me/following/contains?type=user&ids=${usernames}`,
+      {
+        cache: "no-cache",
+      }
+    );
+
+  return response;
+};
```

Comment:
Just directly return the getSpotifyData call, don't save it to a variable and return that.